versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     spotinst = {
       source  = "spotinst/spotinst"
-      version = ">= 1.193.0"
+      version = ">= 1.206.0"
```

Comment:
let's not update the required version, few customers might want to use the older version of provider